fastseq_cli/transformers_generate.py

```diff
@@ -225,6 +227,8 @@ def generate_summaries_or_translations(
                 data_queue.close()
                 msg_queue.close()
                 sys.exit(1)
+            if output_summaries_only:
+                summaries = summaries[:, input_ids.size(1):]
             summaries_cpu = summaries.cpu()
```

Comment:
Nit: will it be better to do the slicing on `summaries_cpu`, which can save gpu memory and may be faster?